library/src/include/logging.h

```diff
@@ -11,55 +11,71 @@
 #include <sys/param.h>
 
 /**
- * @brief Logging function
+ *  * @brief Logging function
```

Comment:
Not sure whether this extra * is interpreted by doxygen.

Reply:
Thanks again,  fixed in next commit